migrations/postgres/20240820035603_db-values.sql

```diff
@@ -0,0 +1,16 @@
+-- +goose Up
+-- create "values" table
+CREATE TABLE "values" (
+  "env_id" uuid NOT NULL,
+  "key" text NOT NULL,
```

Comment:
I guess you need a primary key on (env_id, key) to make it a unique tuple
you can't have two keys with the same name under the same env_id

am I missing something?

Reply:
true. i also want to support scanning all keys for a specific env. how do you suggest i do it?

Reply:
depends on design, 
if there are no key duplicates then
key is string and value is json, so just get the value of this key and you have a the full list

I think you can paginate json array using ```jsonb_array_elements```

if there could be multiple key with the same string, then just standard select